CorsixTH/Lua/dialogs/fullscreen/fax.lua

```diff
@@ -216,10 +216,10 @@ function UIFax:choice(choice_number)
       local campaign_info = self.ui.app.world.campaign_info
       for i, level in ipairs(campaign_info.levels) do
         if self.ui.app.world.map.level_filename == level then
-          local level_info, _ = self.ui.app:readLevelFile(campaign_info.levels[i + 1])
+          local level_info, _ = self.ui.app:readLevelFile(campaign_info.levels[i + 1], campaign_info.folder)
```

Comment:
Existing campaigns will be missing this folder, no?

Reply:
Yes, but until the previous campaigns PR, they were all in the two folders that are checked after that parameter.